tools/shared.py

```diff
@@ -577,7 +577,8 @@ def check_sanity(force=False):
       return # config stored directly in EM_CONFIG => skip sanity checks
     expected = generate_sanity()
 
-    sanity_file = Cache.get_path('sanity.txt')
+    temp_cache = cache.Cache(CACHE, use_subdir=False)
+    sanity_file = temp_cache.get_path('sanity.txt')
```

Comment:
please add a comment explaining why the file is there. I am actually unsure of why that is. what was the problem with the old location?

Reply:
Oh, if it's just that it logically can go up, then that sounds fine.